RestaurantReservationSystem/RestaurantReservationSystem/Controllers/MenuItemController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.Contexts;
+using RestaurantReservation.Db.Models.MenuItem;
+
+namespace RestaurantReservationSystem.Controllers;
+
+[ApiController]
+[Route("api/MenuItems")]
```

Comment:
Usually in url's we make spaces with dashes, so

```suggestion
[Route("api/menu-items")]
```